keystone_scim/contrib/scim/core.py

```diff
@@ -18,7 +18,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-from keystone.common import extension
+#from keystone.common import extension
```

Comment:
Leftover?

Reply:
fixed in https://github.com/telefonicaid/fiware-keystone-scim/pull/63/commits/4823e20f7cb3887260bc88b7f2591b2bfd31dda6